k8s/production/custom/protected-publish/cron-jobs.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: protected-publish
+  namespace: custom
+spec:
+  schedule: "0 */3 * * *" # At the start of every 3rd hour
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        spec:
+          serviceAccountName: protected-publish
+          restartPolicy: Never
+          containers:
+          - name: protected-publish
+            image: ghcr.io/spack/protected-publish:0.0.1
+            imagePullPolicy: IfNotPresent
+            resources:
+              requests:
+                cpu: 1500m
+                memory: 2G
+                ephemeral-storage: "50G"
+            envFrom:
+              - configMapRef:
+                  name: python-scripts-sentry-config
+            args:
+              - "--bucket"
+              - "spack-binaries"
+              - "--ref"
+              - "recent"
+              - "--parallel"
+              - "16"
+            volumeMounts:
+              - name: ephemeral
+                mountPath: "/tmp"
+            env:
+              # Tell tempfile.mkdtemp to use the ephemeral storage
+              #     https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp
+              - name: TMPDIR
+                value: "/tmp"
+          volumes:
+            - name: ephemeral
+              emptyDir:
+                sizeLimit: 50G
+          nodeSelector:
+            spack.io/node-pool: base
```

Comment:
Noting here that currently the biggest instance in our base pool is t3.medium which has 2 vCPU and 4 GiB of memory. I think we'll be fine but it's never quite clear to me ahead of time how much overhead kubelet claims for itself.

Reply:
Yeah, I agree.  I picked `1500m` because I noticed that spackbot worker picks the same when using the base node pool.  :crossed_fingers: